starter-code/src/viking.js

```diff
@@ -1,11 +1,120 @@
-// Soldier
-function Soldier() {}
 
-// Viking
-function Viking() {}
 
-// Saxon
-function Saxon() {}
+//console.log("SECCION SOLDIER");
+
+// Definimos Soldier como una CLASE con sus valores
+
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+}
+// Definimos las funciones de la clase Soldier
+
+Soldier.prototype.attack = function () {
+    return this.strength;
+}
+Soldier.prototype.receiveDamage = function (damage) {
+    this.health -= 50;
+}
+// Definimos Vikings como una instancia de la clase Soldier.Añadimos una propiedad más "name"
+
+function Viking(health, strength) {
+    this.name = "Harold";
+    Soldier.call(this, health, strength);
+}
+
```

Comment:
function Viking(name, health, strength) {
this.name = name;
...........................................